tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/VectorPacket.cs

```diff
@@ -85,16 +85,16 @@ public VectorPacket256 Transpose()
         var r14 = Shuffle(ryz, rxy, 3 << 6 | 1 << 4 | 2 << 2 | 0);
         var r25 = Shuffle(rzx, ryz, 3 << 6 | 1 << 4 | 3 << 2 | 1);
 
-        var m0 = GetLowerHalf<float>(r03);
-        var m1 = GetLowerHalf<float>(r14);
-        var m2 = GetLowerHalf<float>(r25);
+        var m0 = r03.GetLower();
+        var m1 = r14.GetLower();
+        var m2 = r25.GetLower();
         var m3 = ExtractVector128(r03, 1);
         var m4 = ExtractVector128(r14, 1);
         var m5 = ExtractVector128(r25, 1);
 
-        var _Xs = SetHighLow(m1, m0);
-        var _ys = SetHighLow(m3, m2);
-        var _Zs = SetHighLow(m5, m4);
+        var _Xs = Vector256.Create(m0, m1);
```

Comment:
Same with `Create` vs `SetHighLow`, it takes lower then upper; rather than upper then lower